docs/triggerscope-set-up.md

```diff
@@ -0,0 +1,61 @@
+# Setting up the Triggerscope
+
+This guide will walk through setting up the triggerscope (DAC device) to externally drive the LCs for hardware sequencing.
+
+---
+## Table of Contents
+- [Setting up the Triggerscope](#setting-up-the-triggerscope)
+  - [Table of Contents](#table-of-contents)
+  - [Triggerscope](#triggerscope)
+  - [Setup the Triggerscope with micromanager](#setup-the-triggerscope-with-micromanager)
+    - [Plugin the triggerscope](#plugin-the-triggerscope)
+    - [Remove/Check the Meadowlark device is not in config file](#removecheck-the-meadolwark-device-is-not-in-config-file)
+  - [Connect the Meadowlark LC control box to the Triggerscope](#connect-the-meadowlark-lc-control-box-to-the-triggerscope)
+  - [Set the LCs to `external mode`](#set-the-lcs-to-external-mode)
+  - [Launch micromanager with the new config file](#launch-micromanager-with-the-new-config-file)
+  - [FAQ / Troubleshooting](#faq--troubleshooting)
+
+---
+## Triggerscope
+The Triggerscope is ...
+The triggerscope replaces the Meadowlark control box and allows one to drive the LCs through external analog inputs. 
+
+Check the Triggerscope Firmware
+** references to triggerscope setup
+
+## Setup the Triggerscope with micromanager
+** Confirm instructions on setup.
+
+Launch Micromanager, open `Devices > Hardware Configuration Wizard...`, and add the `Triggerscope  Hub` device to your configuration.
+
+Confirm your installation by opening Devices > Device Property Browser... and confirming that `Triggerscope DAC` properties appear.
+
+
+### Plugin the triggerscope
+- Connect the triggerscope via USB
+- Connect the external power supply to the triggerscope
+- Flip on the switch
+
+### Remove/Check the Meadowlark device is not in config file
```

Comment:
Is this necessary? I think there's a Triggerscope + Meadowlark config file on automaton, but I haven't tested it thoroughly.